posts/2023/11/27/nader-podcast.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: Our Executive Director talks with Ralph Nader
+date: "2023-11-27"
+updated: "2023-11-27"
```

Comment:
We don't need the updated date, so let's remove that, and I think we'll be good!